docs/config/configuring-authentication.rst

```diff
@@ -42,25 +42,25 @@ The Flocker CLI package includes the ``flocker-ca`` tool that is used to generat
    - It should be a valid DNS name that HTTPS clients can resolve since they will use it as part of TLS validation.
    - Using an IP address is not recommended as it may break some HTTPS clients.
 
-     .. code-block:: console
+     .. prompt:: bash $
 
-        $ flocker-ca create-control-certificate example.org
+        flocker-ca create-control-certificate example.org
 
 #. At this point you will need to create a :file:`/etc/flocker` directory on each node:
 
-   .. code-block:: console
+   .. prompt:: bash root@centos-7:~/$
 
```

Comment:
should this be 

```
root@centos-7:~/$
```
 or simply

```
bash root@linuxbox:~/# 
```

1) I couldn't see any reference to CentOS earlier in the page, 
2) prompts that end in '$' as a convention mean a user prompt, where prompts that end in '#' mean a root prompt